### PR TITLE
EXP-4 - Adding database dump file

### PR DESCRIPTION
**Pull request by zsoltibv**
Add database dump file with table structures and relations.

### CODE REVIEW

**Review by en-alex-trunchi — APPROVED** [review]
Looks good, I'm not sure if we need all of the generated comments, but we can leave them for now

File: src/main/resources/database_dump/mydb_dump.sql [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+-- Table structure for table `expenses`
+--
+
+DROP TABLE IF EXISTS `expenses`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `expenses` (
+  `expense_id` int NOT NULL AUTO_INCREMENT,
+  `description` varchar(45) DEFAULT NULL,
+  `amount` decimal(10,0) NOT NULL,
```
Comment (en-ciprian-parvan) [inline src/main/resources/database_dump/mydb_dump.sql:81]: We should be able to add values which have 2 decimals.

File: src/main/resources/database_dump/mydb_dump.sql [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  `currency_id` int NOT NULL,
+  `category_id` int NOT NULL,
+  `expense_date` datetime NOT NULL,
```
Comment (en-ciprian-parvan) [inline src/main/resources/database_dump/mydb_dump.sql:84]: This field should have the current datetime as default value.

File: src/main/resources/database_dump/mydb_dump.sql [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+-- Table structure for table `users`
+--
+
+DROP TABLE IF EXISTS `users`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `users` (
+  `user_id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  `created_at` timestamp NOT NULL,
```
Comment (en-ciprian-parvan) [inline src/main/resources/database_dump/mydb_dump.sql:116]: For consistency, please change the field type to ```datetime``` and add current datetime as default value also.

**Review by en-ciprian-parvan — CHANGES_REQUESTED** [review]

**Review by en-ciprian-parvan — APPROVED** [review]

**Pull request merged**